source/diagrams/manage-diagrams/rename-diagrams.txt

```diff
@@ -10,4 +10,47 @@ Rename Diagrams
    :depth: 1
    :class: singlecol
 
-Rename diagrams.
\ No newline at end of file
+You can rename diagrams on the Mapping tab of Relational Migrator.
+Relational Migrator displays diagram names on the ribbon tab. 
+
+About this Task
+---------------
+
+You can rename diagrams to customize your diagram properties. Relational
+Migrator shows your diagram names on the ribbon tab of the Mapping view and in
+the Diagram Properties pane. 
+
+Steps
+-----
+
+.. procedure::
+   :style:  connected
+
+   .. step:: Navigate to the diagram you want to rename. 
+
+      Click the name of the diagram in the ribbon tab to display the
+      diagram. 
+   
+   .. step:: Open the Diagram Properties pane.
+
+      Click the arrow button on the right of your window to open the
+      Diagram Properties pane. 
```

Comment:
[style guide] Similar comment as the one above about the :guilabel: directive!
```suggestion
      :guilabel:`Diagram Properties` pane. 
```

Reply:
Done!